config/config.go

```diff
@@ -33,15 +39,32 @@ type IngressConfig struct {
 func Get() *IngressConfig {
 
 	options := viper.New()
-	options.SetDefault("MaxSize", 10*1024*1024)
-	options.SetDefault("Port", 3000)
-	options.SetDefault("StageBucket", "available")
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
+		cfg := clowder.LoadedConfig
+		topic, ok := clowder.KafkaTopics["platform.upload.advisor"]
+		if !ok {
+			log.Fatal("topic not found")
+		}
+		options.SetDefault("Port", cfg.WebPort)
+		options.SetDefault("KafkaBrokers", fmt.Sprintf("%s:%v", cfg.Kafka.Brokers[0].Hostname, *cfg.Kafka.Brokers[0].Port))
+		options.SetDefault("KafkaGroupID", cfg.Kafka.Topics[0].ConsumerGroup)
```

Comment:
I don't think clowder actually populates consumer group.  Probably should make sure that this is provided by the app for now.

Reply:
also, if we add platform.payload-status to the list of requested topics, then we will need to look up topics by `requestedName`